src/pages/Content/index.js

```diff
@@ -28,6 +28,8 @@ chrome.runtime.onMessage.addListener(function (msgObj) {
       });
       break;
     case 'startPlaying':
+      // Play the records from the Popup
+      recordedEvents = JSON.parse(msgObj.value);
```

Comment:
Create a new var (no need to touch the global var):
```suggestion
      const eventsToPlay = JSON.parse(msgObj.value);
```